build-bench-env.sh

```diff
@@ -565,11 +565,12 @@ if test "$setup_dh" = "1"; then
   checkout dh $version_dh https://github.com/emeryberger/DieHard
   # remove all the historical useless junk
   rm -rf ./benchmarks/ ./src/archipelago/ ./src/build/ ./src/exterminator/ ./src/local/ ./src/original-diehard/ ./src/replicated/ ./docs
-  if test "$darwin" = "1"; then
-    TARGET=libdieharder make -C src macos
-  else
-    TARGET=libdieharder make -C src linux-gcc-64
-  fi
+  cd src
+  mkdir -p build
+  cd build
+  cmake ..
+  make
```

Comment:
```suggestion
  cmake -S . -B build
  cmake --build build -j `nproc`
```

I think this is how it's done with modern cmake